Core/Core/Files/FileDetails/FileDetailsViewController.swift

```diff
@@ -351,32 +351,29 @@ extension FileDetailsViewController: PSPDFViewControllerDelegate {
 
     func saveAnnotations() {
         for child in children {
-            _ = try? (child as? PSPDFViewController)?.document?.save()
+            _ = try? (child as? PDFViewController)?.document?.save()
         }
     }
 
-    public func pdfViewController(
-        _ pdfController: PSPDFViewController,
-        shouldShow menuItems: [PSPDFMenuItem],
-        atSuggestedTargetRect rect: CGRect,
-        for annotations: [PSPDFAnnotation]?,
-        in annotationRect: CGRect,
-        on pageView: PSPDFPageView
-    ) -> [PSPDFMenuItem] {
+    public func pdfViewController(_ pdfController: PDFViewController,
+                                  shouldShow menuItems: [MenuItem],
+                                  atSuggestedTargetRect rect: CGRect,
+                                  forSelectedText selectedText: String,
+                                  in textRect: CGRect, on pageView: PDFPageView) -> [MenuItem] {
```

Comment:
Bad indent. Also the options shown when you select an ink annotation have regressed. It is showing "Inspector" instead of "Style", "Note" should be hidden, and the delete icon should be using our own trash icon. This is the function that used to make this happen. I'm guessing the "forSelectedText" is a different delegate method than the "for annotations" this used to be.